### ISSUE TITLE
Feature Request: thematic auto-ping with cooldown timer

### ISSUE THREAD

**Issue by Zitchas** [EndlessSkyCommunity/EndlessSky-Discord-Bot#156]
What I would like to do is establish a mechanism for having an auto-ping related to a specific topic, but with a timer built into it such that it won't trigger more than once per designated time limit.

As a bonus feature, it would be useful to limit it to a specific channel.

For example, I would like to be pinged when someone discusses the Remnant in the bug reporting channel. In order to avoid cluttering the channel, however, I would prefer if - once the auto-ping has been triggered - it wouldn't ping for another hour. Thus if there's a raging discussion on the topic, the conversation isn't going to get filled up with auto-pings.

Thank you!

**Comment by MCOfficer** [comment]
this can be implemented in pretty much the same way as the old meme command, which was removed in #144. it would require some more specific filters (channel & cooldown), of course.

**Comment by tehhowch** [comment]
Or you get a PM with a ink to the message. No cooldown needed

**Comment by Zitchas** [comment]
That would work. Would it be possible to ommit myself?

**Comment by tehhowch** [comment]
Yep. I believe I created an issue for this type of ping already, too
